### ISSUE TITLE
Tax-Calculator pull requests #800 and #801

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/Tax-Calculator#802]
In order for pull request #801 to work correctly, the changes in pull request #800 need to be merged first.

If you have any comments or concerns regarding either pull request, please voice them now (that is, Thursday afternoon).  Otherwise, these two pull requests will be merged into the master branch on Friday.

@MattHJensen @feenberg @talumbau @zrisher @Amy-Xu @GoFroggyRun

**Comment by martinholmer** [comment]
Issue #802 resolved by merger of pull requests #800 and #801.
